bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseThingHandler.java

```diff
@@ -174,6 +178,18 @@ private void updateHANFUNAlarmSensor(@Nullable AlertModel alertModel) {
         }
     }
 
+    private void updateHANFUNBlindsAlarmSensor(@Nullable AlertModel alertModel) {
+        if (alertModel != null) {
+            updateThingChannelState(CHANNEL_OBSTRUCTION_ALARM,
+                    alertModel.hasObstructionAlarmOccurred() ? OnOffType.ON : OnOffType.OFF);
+            updateThingChannelState(CHANNEL_TEMPERATURE_ALARM,
+                    alertModel.hasTemperaturAlarmOccurred() ? OnOffType.ON : OnOffType.OFF);
```

Comment:
You can rather use: `OnOffType.from(boolean)` like for example: `updateThingChannelState(CHANNEL_TEMPERATURE_ALARM, OnOffType.from(alertModel.hasTemperaturAlarmOccurred()));`